deposit/src/main/java/edu/unc/lib/deposit/validate/ValidateFileAvailabilityJob.java

```diff
@@ -104,4 +96,13 @@ public void runJob() {
 		}
 	}
 
+	private void addLocations(Set<String> hrefs, Model model, Property property) {
+		Property fileLocation = CdrDeposit.stagingLocation;
+		StmtIterator i = model.listStatements(new SimpleSelector((Resource) null, fileLocation, (RDFNode) null));
```

Comment:
yeah, othewise it won't compile since it can't determine which overridden version of the method to use